src/js/index.js

```diff
@@ -276,3 +276,24 @@ $(document).ready(() => {
     }
   })
 })
+
+/**
+ * Fetch status from statuspage.com and update link in footer
+ */
+$(document).ready(() => {
+  fetch('https://nq8lf8mrmvw6.statuspage.io/api/v2/status.json')
```

Comment:
BTW this is a nice feature for our status page

Reply:
@maxigimenez wadda ya mean?

Reply:
expose a json endpoint to easily grab the overall status, so customer can easily add indicators anywhere like we are doing